src/components/_common/headers/themed-page-header/ThemedPageHeader.less

```diff
@@ -20,26 +19,15 @@
     }
 
     &__illustration {
-        width: #px-to-rem(96px) [];
-        height: #px-to-rem(96px) [];
-        margin-right: 1.5rem;
+        width: #px-to-rem(80px) [];
+        height: #px-to-rem(80px) [];
+        margin-right: 2rem;
         pointer-events: none;
     }
 
     &__text {
         width: 100%;
-        max-width: 56.5rem;
-
-        .page-header {
-            padding: 0;
-            h1 {
-                .typo-innholdstittel;
-
-                @media @mq-screen-tablet-and-desktop {
-                    .typo-sidetittel;
-                }
```

Comment:
Dette var ment for å skalere tittel-teksten ned på mobil, må ha inn en alternativ løsning her. Ser forøvrig at de nye title-klassene ikke har noen responsiv skalering i det hele tatt, så problemet med for stor tittel på mobil blir verre enn tidligere 😄